src/display/api.js

```diff
@@ -1590,9 +1583,15 @@ const PDFWorker = (function PDFWorkerClosure() {
   }
 
   /**
-   * @param {PDFWorkerParameters} params - The worker initialization parameters.
+   * The web worker abstraction controls the instantiation of PDF documents.
```

Comment:
Please keep the `PDF.js`-part of this line, perhaps as `PDF.js web worker abstraction, which controls the instantiation of PDF documents.`, since this is different from the general concept of the [Web Workers API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API).

Reply:
Done. Thanks!